### ISSUE TITLE
Very cool project. A little advice

### ISSUE THREAD

**Issue by MK85Pilot** [vsaint1/cheetos-engine#2]
Interesting release, but local deployment of web takes a lot of resources, can you let external programs run locally, web in vps

**Comment by isvktt** [comment]
What steps did you take to open it?

**Comment by croseL55** [comment]
> Interesting release, but local deployment of web takes a lot of resources, can you let external programs run locally, web in vps

yes u can, but its same takes a lot of resources. i have try it using web vps on my own project
